tests/core/unittests/mongodb_test.py

```diff
@@ -47,10 +36,12 @@ def clean_db(database, exp_config):
 @pytest.fixture(scope='module')
 def moptdb():
     """Return MongoDB wrapper instance initiated with test opts."""
+    MongoDB.instance = None
```

Comment:
Is it needed here because ```moptdb()``` is not a test so you cannot use ```null_db_instances``` fixture?

Reply:
The problem is that `null_db_instances` has `'function'` scope, while `moptdb` has `'module'` scope. If there's a need to create a `'module'` scope one I will do it, but currently there's no other needs for it. (It's kinda annoying, I know..)

Reply:
ok. Thanks for the explanation.